tardis/io/util.py

```diff
@@ -160,7 +211,7 @@ def traverse_configs(base, other, func, *args):
             traverse_configs(base[k], other[k], func, *args)
     elif (
         isinstance(base, collections_abc.Iterable)
-        and not isinstance(base, basestring)
```

Comment:
why is this changed?

Reply:
the change was made to ensure that function is compatible with python 3